### ISSUE TITLE
Fails to boot when using the -lto/clang build v3 cachyos kernel (all versions)

### ISSUE THREAD

**Issue by LethalManBoob** [CachyOS/linux-cachyos#34]
[boot.txt](https://github.com/CachyOS/linux-cachyos/files/9679221/boot.txt)
The boot screen hangs on initializing graphics device or somthing like that. It then goes to a black screen where i must alt del f3 and reboot. 
I run an nvidia rtx 3070 and i use drm mode. If more information is needed I will give it.

**Comment by sirlucjan** [comment]
Did you install clang, llvm and lld before run dkms?

**Comment by ptr1337** [comment]
Since which kernel version this is happening ?
Are GCC Kernels working fine ?
When you self compile a kernel with LTO, does it work? 
Could you try linux-cachyos-rc-lto linux-cachyos-rc-lto-headers ?
Did dkms did compiled the headers without a error ?
Actually I dont see anything special in this dmesg log. Seems so far fine to me. 

The only thing I currently see, which is different to my dmesg log (have also a nvidia card), that you dont have any NVRM entrys.
I think the dkms modules have failed, through a missing clang,lld,llvm.

**Comment by ptr1337** [comment]
Actually at my logs:
```
[    2.881402] NVRM: loading NVIDIA UNIX x86_64 Kernel Module  515.76  Mon Sep 12 19:21:56 UTC 2022
[    2.898635] nvidia-modeset: Loading NVIDIA Kernel Mode Setting Driver for UNIX platforms  515.76  Mon Sep 12 19:11:54 UTC 2022
```

**Comment by sirlucjan** [comment]
We have added clang, llvm and lld as dkms depends:

https://github.com/CachyOS/CachyOS-PKGBUILDS/commit/73ab999dfa3ba7aea47a0c4d2ab576119f2663b6

Probably you have to install clang, llvm and lld  and run dkms once again.

**Comment by ptr1337** [comment]
@LethalManBoob 

I think this can be closed and has been fixed with 6.0 ?

**Comment by LethalManBoob** [comment]
> @LethalManBoob
> 
> I think this can be closed and has been fixed with 6.0 ?

Yes. This issue has been resolved.
